drivers/gpu/drm/v3d/v3d_debugfs.c

```diff
@@ -197,11 +197,11 @@ static int v3d_v3d_debugfs_ident(struct seq_file *m, void *unused)
 		seq_printf(m, "  QPUs:         %d\n", nslc * qups);
 		seq_printf(m, "  Semaphores:   %d\n",
 			   V3D_GET_FIELD(ident1, V3D_IDENT1_NSEM));
-		if (v3d->ver <= 42) {
+		if (v3d->ver <= V3D_GEN_42) {
 			seq_printf(m, "  BCG int:      %d\n",
 				   (ident2 & V3D_IDENT2_BCG_INT) != 0);
 		}
-		if (v3d->ver < 40) {
+		if (v3d->ver < V3D_GEN_41) {
```

Comment:
Is this correct? We are still setting v3d->ver based on register values, and if V3D_GEN_41 == 41 and there is hardware with a version of 40 then this test has changed.

Reply:
It is correct. V3D 4.0 doesn't exist and hypothetically, if it did, the kernel doesn't support it. The kernel only supports 33, 41, 42, and 71.

Reply:
I'll take your word for it.